frontend/src/components/SearchQuery/SearchQuery.css

```diff
@@ -23,4 +23,4 @@
 
 .hidden {
     display: none;
-}
\ No newline at end of file
+} */
```

Comment:
It's not a good practice to commit commented code. This should be removed, you can always find the previous version of the code with git, or looking at the Pull Requests if you need to recover the code.